Cargo.toml

```diff
@@ -43,7 +43,7 @@ chrono = { version = "0.4.19", features = ["serde"] }
 orion = "0.16.0"
 hex = "0.4.3"
 # tako = { path = "../tako" }
-tako = { git = "https://github.com/spirali/tako", branch = "main" }
+tako = { git = "https://github.com/spirali/tako", branch = "timelimit" }
```

Comment:
Also don't forget to update this before merging.

Reply:
Thank you for reminding :)